Sources/Helpers/OASettingsImporter.mm

```diff
@@ -547,17 +576,18 @@ - (BOOL) doInBackground
 {
     [_importer importItems:_file items:_items];
     
-    NSString *tempDir = [[OsmAndApp instance].documentsPath stringByAppendingPathComponent:@"backup"];
-    [NSFileManager.defaultManager createDirectoryAtPath:tempDir withIntermediateDirectories:YES attributes:nil error:nil];
-    
-    for (OAProfileSettingsItem *item in _items)
-    {
-        NSString *bakupPatch = [[tempDir stringByAppendingPathComponent:item.appMode.stringKey] stringByAppendingPathExtension:@"osf"];
-        if (![[NSFileManager defaultManager] fileExistsAtPath:bakupPatch])
-        {
-            [[NSFileManager defaultManager] copyItemAtPath:_file toPath:bakupPatch error:nil];
-        }
-    }
+    // to check if we need it
+//    NSString *tempDir = [[OsmAndApp instance].documentsPath stringByAppendingPathComponent:@"backup"];
```

Comment:
This code is needed for the profile reset functionality (though I'm not sure why the whole osf is being copied, so please check with the Android version).

Reply:
I have checked with ImportItemsAsyncTask in Android, this code is placed incorrectly, someone needs to port it correctly from Java